aiohttp/test_utils.py

```diff
@@ -110,7 +114,15 @@ def __init__(self,
         await self.runner.setup()
         if not self.port:
             self.port = 0
-        _sock = get_port_socket(self.host, self.port)
+        absolute_host = self.host
+        try:
+            version = ipaddress.ip_address(self.host).version
+            if version == 6:
+                absolute_host = f"[{self.host}]"
+        except ValueError:
+            version = 4
```

Comment:
try/except blocks must always wrap only one line where the exception is expected
```suggestion
        try:
            version = ipaddress.ip_address(self.host).version
        except ValueError:
            version = 4
        if version == 6:
            absolute_host = f"[{self.host}]"
```